src/components/TrendChart.js

```diff
@@ -281,7 +210,17 @@ class TrendChart extends React.Component {
                     definition
                   </text>
                 </g>}
-              {callout}
+              <TrendChartHover
+                color={color}
+                data={places
```

Comment:
for legibility, it feels like this (the `data` value in `TrendChartHover`) should be pulled out into a variable above